example/src/demos/basic-usage.tsx

```diff
@@ -12,7 +12,23 @@ const App = () => {
   const [value, setValue] = useState<CountryPhoneInputValue>({ short: 'US' });
 
   return (
-    <ConfigProvider locale={en}>
+    <ConfigProvider
+      locale={en}
+      areaSorter={(a, b) => {
```

Comment:
Thanks for updating the demo. Since this file only meant for `basic-usage`, it would be better to create a new file like `custom-sorter.tsx`.

Reply:
@helsonxiao I just added the custom-sorter.tsx